.travis.yml

```diff
@@ -1,6 +1,28 @@
+sudo: required
+dist: trusty
```

Comment:
Why do you use trusty ?
It doesnt have a lot of python stuff installed by default, if you use the older one it would be better I think (Trusty is in beta)